Userland/Libraries/LibGfx/ICCProfile.cpp

```diff
@@ -271,6 +289,18 @@ char const* color_space_name(ColorSpace color_space)
     }
 }
 
+char const* profile_connection_space_name(ColorSpace color_space)
```

Comment:
I realize you are just extending the existing pattern in this code, but shouldn't we be returning `StringView`'s here?

Reply:
I just happened across the discussion in https://github.com/SerenityOS/serenity/pull/16724#discussion_r1059401594 but I'm not sure I get your perf argument? We write modern C++ everywhere, and in my mind that means avoiding raw C strings as much as humanly possible. We use `Array<T>` everywhere instead of C/C++ arrays, and I don't see why C-Strings are any different here...

Reply:
How is a C string less type safe than a StringView?

(It's not a perf argument, it's a binary size argument. StringView and `const char*` do different things; both have their uses imho. But you're the third person to ask about it, so it might be less work for me to just change it. On the third hand, this isn't new code in this PR.)

Reply:
> How is a C string less type safe than a StringView?

All indexing / enumeration goes through the operator overload where we force bounds checking. I guess you can debate if this is "type safety" or "memory safety". 

> StringView and const char* do different things; both have their uses imho

Sure, but in Serenity (at least in all the code I've read and remember, maybe I'm missing some context) we favor `StringView` everywhere besides `LibC`. I was curious why this Library is special in that context, why does it need to use (what I believe are) different patterns than the rest of the monorepo? 

> On the third hand, this isn't new code in this PR.

Sure, we can move on with this PR, I agree it's inconsequential to the immediate goals of these change.

Reply:
> All indexing / enumeration goes through the operator overload where we force bounds checking. 

You can call `characters_without_null_termination` on a StringView and deref that just as well. I agree that indexing a `const char*` isn't safe without explicit additional checking, but using it as a string (as used here) seems like it's just as safe.

But sure, I'll switch this file to StringView. Best practices and all.

Reply:
#16745 does this. Up to you which one to merge first; will rebase the other one after.